vertx-sockjs-service-proxy/src/test/resources/vertx-js/vertx-eventbus.js

```diff
@@ -308,6 +340,16 @@
     }
   };
 
+  EventBus.prototype.reconnectEnabled = function (enable) {
```

Comment:
i think the reverse name would be better: `enableReconnect` so it sound more natural when you read the code:
on the `eventbus` will set `enable` `reconnect` to `true` if you know what i mean...